include/stout/tests/utils.hpp

```diff
@@ -69,10 +64,13 @@ class MixinTemporaryDirectoryTest : public T
     Result<std::string> realpath = os::realpath(directory.get());
 
     if (realpath.isError()) {
-      return Error("Failed to get realpath of '" + directory.get() + "'"
-                   ": " + realpath.error());
+      return Error("Failed to get realpath of '" + directory.get() +
+                   "'"
+                   ": "
+                   + realpath.error());
```

Comment:
This seems like a weird reformat that I'm guessing `clang-format` did because these are string continuations. How about instead we add the missing `+`:
```suggestion
      return Error("Failed to get realpath of '" + directory.get() + "'"                   
                   + ": " + realpath.error());
```